README.md

```diff
@@ -1,6 +1,12 @@
-| CicleCI Build Status | Sonar Code Quality | DockerHub Release | DockerHub Release Candidate | DockerHub Develop |
-|------|-----|-----|-----|-----|
-[![CircleCI](https://circleci.com/gh/usdot-fhwa-stol/carma-platform.svg?style=svg)](https://circleci.com/gh/usdot-fhwa-stol/carma-platform) | [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=usdot-fhwa-stol_CARMAPlatform&metric=alert_status)](https://sonarcloud.io/dashboard?id=usdot-fhwa-stol_CARMAPlatform) | [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/usdotfhwastol/carma-platform?label=Usdotfhwa%20STOL&logo=%232496ED)](https://hub.docker.com/repository/docker/usdotfhwastol/carma-platform) | [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/usdotfhwastolcandidate/carma-platform?label=Usdotfhwastol%20Candidate&logo=%232496ED)](https://hub.docker.com/repository/docker/usdotfhwastolcandidate/carma-platform) | [![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/usdotfhwastoldev/carma-platform?label=Usdotfhwastol%20DEV&logo=%232496ED)](https://hub.docker.com/repository/docker/usdotfhwastoldev/carma-platform)
+## Develop CI/CD Processes
+| DockerHub Image Build | Unit Tests Workflow |  Sonar Code Quality |
+|----------------------|---------------------|---------------------|
+ [![Build Workflows](https://github.com/usdot-fhwa-stol/carma-platform/actions/workflows/dockerhub.yml/badge.svg?branch=develop)](https://github.com/usdot-fhwa-stol/carma-platform/actions/workflows/dockerhub.yml) |  [![Build Workflows](https://github.com/usdot-fhwa-stol/carma-platform/actions/workflows/Ci.yml/badge.svg?branch=develop)](https://github.com/usdot-fhwa-stol/carma-platform/actions/workflows/Ci.yml) | [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=usdot-fhwa-stol_CARMAPlatform&metric=alert_status)](https://sonarcloud.io/dashboard?id=usdot-fhwa-stol_CARMAPlatform) |
+ ## Release CI/CD Processes
```

Comment:
I forgot that both our dockerhub.yml and Ci.yml workflows were not functional (i.e. not necessarily that they were failing, but we had not completed the development of the workflows) at the time of our latest release, so this information is not very useful right now.

Let's remove these for now, I'll add a story to the backlog to update these in time for the next CARMA Platform release.